grammars/css.cson

```diff
@@ -483,7 +483,8 @@
         'include': '#comment-block'
       }
       {
-        'match': '--([A-Za-z0-9_-]+)'
+        'match': '--([A-Za-z][A-Za-z0-9_-]*)'
```

Comment:
The pattern is a bit too strict.

From https://www.w3.org/TR/css-variables/#custom-property:

> it’s defined as any valid identifier that starts with two dashes

And a *valid* identifier shares the same syntax as an `<ident-token>`—(escaping set aside) it

  * **may contain only**
    * **[a-zA-Z0-9_-]**
    * **ISO 10646 characters U+00A0 and higher**
  * must not begin with \[0-9]
  * must not begin with '-' followed by \[0-9]
  * must be at least 2 characters long if begins with '-'

Since the identifier in this case always starts with `--`, the last three requirements are fulfilled, you only have to care about the first rule (in bold) here. So a simple

```
--[\\w-]+
```

suffices. Just in case, `\w` is not equivalent to `[a-zA-Z0-9_]` with the Unicode defined encodings like UTF-8. 😉